sw/airborne/firmwares/rotorcraft/navigation.h

```diff
@@ -212,6 +222,9 @@ bool nav_check_wp_time(struct EnuCoor_i *wp, uint16_t stay_time);
   }
 
 #define NavSetManual nav_set_manual
+#define NavSetFailsafe { \
```

Comment:
Would be better as `#define NavSetFailsafe nav_set_failsafe`
Or even better we start to get rid of the macros and use the functions directly...